### DISCUSSION TITLE
rtsp wont start on Ubuntu 20.04 lts

### DISCUSSION THREAD

**Discussion by Marshmellow84** [bluenviron/mediamtx#1091] [General]
i am using rtsp the arm64v8

I keep getting these message

2022/08/15 20:51:07 INF rtsp-simple-server v0.17.17
2022/08/15 20:51:07 WAR configuration file not found, using an empty configuration
2022/08/15 20:51:07 ERR listen udp :8000: bind: address already in use
2022/08/15 20:51:07 INF waiting for external commands

I have moved files to

rtsp-simple-server /usr/local/bin/

rtsp-simple-server.yml /usr/local/etc/

I am not sure why the configuration file is not found

I also ran this

sudo tee /etc/systemd/system/rtsp-simple-server.service >/dev/null << EOF
[Unit]
After=network.target
[Service]
ExecStart=/usr/local/bin/rtsp-simple-server
/usr/local/etc/rtsp-simple-server.yml
[Install]
WantedBy=multi-user.target
EOF

any help would be great, i been at this for hours now

**Comment by rapi3** [comment]
Port 8000 you try to use it is already used by another app... change the port or disable it if you don't need.
```
...
2022/08/15 20:51:07 ERR listen udp :8000: bind: address already in use
...
```

**Reply by Marshmellow84** [reply]
Oh my word. So it is, that's what I get for a long day. Silly mistake on my part. Ty for that, ill look at the yml file and change that

[a comment by an automated account is not shown]
